experimental/packages/opentelemetry-api-metrics/src/NoopMeter.ts

```diff
@@ -39,87 +39,87 @@ export class NoopMeter implements Meter {
   /**
    * @see {@link Meter.createHistogram}
    */
-  createHistogram(_name: string, _options?: MetricOptions): Histogram {
-    return NOOP_HISTOGRAM_METRIC;
+  createHistogram<AttributesTypes extends MetricAttributes=MetricAttributes>(_name: string, _options?: MetricOptions): Histogram<AttributesTypes> {
```

Comment:
TBO, I don't think updates to Noop types are needed. People should use the API types instead of using Noop types directly.

Noop types are considered as a default "SDK" implementation. We need to make sure SDK is not required to provide such type parameters.